bootstrap/absolute.sh

```diff
@@ -47,4 +47,9 @@ done
 # TODO(nadongguri): this env vir should be set not here but before gulp command
 export BABEL_CACHE_PATH=$(absolute_path)/.babel-cache.json
 
-gulp $@
+# ccl requires several steps. it need to turn off the gulp message
+if [ "$@" = "cc" ]; then
+  gulp $@ --silent
```

Comment:
As far as I heard, putting silent option is to avoid async call sequence. But is there other ways to handle?